### ISSUE TITLE
creating a release with v2 cli fails after creating release with v1 cli

### ISSUE THREAD

**Issue by abbyachau** [cloudfoundry/bosh-cli#145]
We often use routing-release repo to generate dev releases using v1 cli; `bosh create release`. 

Now we're using the cf-deployment manifest, which includes routing release to provide source for gorouter. When we set `version: create` and deploy using v2 cli, generating dev release on the fly, the deploy fails with mysterious error:

```
21:02:54 | Error: Action Failed get_task: Task 0440f4c4-ac01-4f16-73d2-1cbdb118f537 result: Compiling package golang1.7: Running packaging script: Running packaging script: Command exited with 2; Stdout: , Stderr: + VERSION=1.7.4
```

After consulting with @cppforlife, the solution seemed to be some combination of `bosh2 reset-release` and `bosh2 remove-blob`.

The issue was apparently that blobs downloaded by the v1 cli cannot be reused by the v2 cli or cf-deployment manifest? We would like to see either:
- v2 cli able to use or replace blobs downloaded by v1 cli
- A better error message that clearly indicates how to solve the problem, or at least what the real problem is.

**Comment by tejaycar** [comment]
+1 we have the same problem.  My blobs are being deleted from the blobs dir when I try to build a --force release with cli2

**Comment by idoru** [comment]
We encountered this problem today and wasted a lot of time tracking it down. We finally noticed it when we unpacked the release and found that the blobs were all symbolic links.

It'd be great to have this process fail fast; i.e. the v2 CLI ought to inform at the time of `create-release` that the blobs previously downloaded by the v1 CLI are incompatible.

**Comment by Kiemes** [comment]
@idoru There is already a PR for it https://github.com/cloudfoundry/bosh-cli/pull/259. Let's hope this gets merged soon and will lead to a new CLI version

**Comment by tjvman** [comment]
PR has been merged and its functionality is available as of version 2.0.30. Closing.
